components/prism-hdl.js

```diff
@@ -0,0 +1,18 @@
+Prism.languages.hdl = {
+	'comment': [
+		{
+			pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
```

Comment:
You probaböy borrowed this from C-like, but this language isn't as complex as C, C++, etc., so we can simplify and combine the two comment patterns to this:

```js
/\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/
```

No `lookbehind` or `greedy` required.